package.json

```diff
@@ -32,5 +32,8 @@
   "publishConfig": {
     "access": "public",
     "provenance": true
+  },
+  "devDependencies": {
+    "eslint-plugin-distributive": "github:Distributive-Network/eslint-plugin-distributive"
```

Comment:
**issue:** This won't be picked up by consumers of the config.

ESLint plugins need to be declared as peer dependencies, as per the [docs](https://eslint.org/docs/latest/extend/shareable-configs#publishing-a-shareable-config):

> If your shareable config depends on a plugin, you should also specify it as a peerDependency (plugins will be loaded relative to the end user’s project, so the end user is required to install the plugins they need).

I'd recommend testing the use of the shared config locally first before pushing new changes.